application/src/main/java/com/softserve/lv460/application/controller/UserApplicationController.java

```diff
@@ -0,0 +1,64 @@
+package com.softserve.lv460.application.controller;
+
+import com.softserve.lv460.application.entity.ApplicationUser;
+import com.softserve.lv460.application.repository.ApplicationUserRepository;
+import com.softserve.lv460.application.security.dto.JWTUserRequest;
+import com.softserve.lv460.application.security.dto.JWTUserResponse;
+import com.softserve.lv460.application.exceptions.UserAlreadyRegisteredException;
+import com.softserve.lv460.application.security.jwt.JwtTokenProvider;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/users")
+public class UserApplicationController {
+  private final ApplicationUserRepository applicationUserRepository;
```

Comment:
All logic should be in service class instead of controller. Create service class